documentation/src/components/side-nav-search.ts

```diff
@@ -87,7 +94,7 @@ class SearchComponent extends LitElement {
                                                 this.closePopover();
                                             }}
                                         >
-                                            ${result.name}
+                                            ${result.label}
```

Comment:
Use `Action Button` instead of `action-button` in search results.

Reply:
👍